Makefile

```diff
@@ -42,6 +42,10 @@ HTTPS_PROXY_URL := $(or $(HTTPS_PROXY_URL), "")
 NO_PROXY_VALUES := $(or $(NO_PROXY_VALUES), "")
 VIP_DHCP_ALLOCATION := $(or $(VIP_DHCP_ALLOCATION),yes)
 
+#day2 params
+API_VIP_IP := $(or $(API_VIP_IP),"")
```

Comment:
why not to take them from cluster?

Reply:
the whole point of day2 is that we don't have an access to the OCP cluster. The way i want to run this is that we first run OCP cluster full install and then we run a target of day2.

Reply:
right. But we have. So why test will not just take from cluster. I think it is bad idea to ask user to remember his vips.
Not need in test env

Reply:
but maybe i am wrong